### ISSUE TITLE
416.94 nvidia driver required - 397.44 detected - Battlefield V

### ISSUE THREAD

**Issue by jirir** [williamparry/cloudRIG#119]
Battlefield V won't run on cloudRIG because:

Detected NVIDIA GeForce driver version 397.44. The required driver version is 416.94 or later.
...
![IMG_3306](https://user-images.githubusercontent.com/20031425/55670675-bf033b00-5887-11e9-8107-c0b0743aa218.jpg)

**Comment by Banginpadr** [comment]
> Battlefield V won't run on cloudRIG because:
> 
> Detected NVIDIA GeForce driver version 397.44. The required driver version is 416.94 or later.
> ...
> ![IMG_3306](https://user-images.githubusercontent.com/20031425/55670675-bf033b00-5887-11e9-8107-c0b0743aa218.jpg)
> 
> There is a **[$20 open bounty](https://www.bountysource.com/issues/72478118-416-94-nvidia-driver-required-397-44-detected-battlefield-v?utm_campaign=plugin&utm_content=tracker%2F51879914&utm_medium=issues&utm_source=github)** on this issue. Add to the bounty at [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F51879914&utm_medium=issues&utm_source=github).

what is that you need the windows drivers for you pc?

**Comment by jirir** [comment]
With the current nvidia drivers in the cloudRig PC (based on parsec AMI) - Battlefield V does not run because it says it needs newer drivers - see the screenshot with the error message.

**Comment by Banginpadr** [comment]
> With the current nvidia drivers in the cloudRig PC (based on parsec AMI) - Battlefield V does not run because it says it needs newer drivers - see the screenshot with the error message.

yeah i know, i was having the same issue months ago so i just update it and now everything is fine

**Comment by jirir** [comment]
How did you update the nvidia driver in the g3/g3s instance?

The parsec site says that it is not possible to update the video drivers in the g3 instances.

Of course I’ve tried and hadn’t succeeded.

So this looks like good news.

**Comment by Banginpadr** [comment]
> How did you update the nvidia driver in the g3/g3s instance?
> 
> The parsec site says that it is not possible to update the video drivers in the g3 instances.
> 
> Of course I’ve tried and hadn’t succeeded.
> 
> So this looks like good news.

i meant locally and i was using a mac but i dont think windows will be any different

**Comment by mKeRix** [comment]
Fyi, this script should be able to solve your problems: https://github.com/jamesstringerparsec/Cloud-GPU-Updater

It says to refrain from doing this, but it updated my drivers just fine. If you kill your machine you can always recreate it anyway.
